src/server.lisp

```diff
@@ -238,18 +239,20 @@ DISPATCH-TABLE and LOGGING-STREAM are both required arguments.  TIMEOUT is of ty
                                                "Request ~a error: Unhandled error in host program:~%~a"
                                                (|RPCRequest-id| request)
                                                c)
+                       (when debug
+                         (cl-syslog:format-log logger ':err (trivial-backtrace:print-backtrace c :output nil)))
```

Comment:
Upon re-reading the bug description for #87, I might have misinterpreted. On the first read, I thought it was asking for logging of backtraces, but maybe it's asking to tack the backtrace onto the `|RPCError|` reply some how?

@notmgsk is this what you had in mind?

another option would be to just re-signal the error so you wind up in the debugger for interactive use.

Reply:
welp, just realized this is pretty useless since the stack has already unwound by the time I log the backtrace... not my best PR... closing for now.